AEPLifecycle/Sources/LifecycleV2/LifecycleV2MetricsBuilder.swift

```diff
@@ -164,3 +164,26 @@ class LifecycleV2MetricsBuilder {
     }
 
 }
+
+extension SystemInfoService {
+
+    /// Return a string for the given 'locale' identifier.
+    /// Uses the format "`Locale.languageCode`-`Locale.regionCode`".
+    /// The default locale is 'Locale.autoupdatingCurrent'.
```

Comment:
docs: for this line I suggest you rather defer to SystemInfoService getActiveLocaleName docs